includes/gateway/class-omise-payment-billpayment-tesco.php

```diff
@@ -137,84 +137,92 @@ public function display_barcode( $order, $context = 'view' ) {
 				$charge['source']['references']['reference_number_2'],
 				$charge['amount']
 			);
+			$expires_datetime   = new WC_DateTime( $charge['source']['references']['expires_at'], new DateTimeZone( 'UTC' ) );
+			$expires_datetime->set_utc_offset( wc_timezone_offset() );
 			?>
 
 			<div class="omise omise-billpayment-tesco-details" <?php echo 'email' === $context ? 'style="margin-bottom: 4em; text-align:center;"' : ''; ?>>
```

Comment:
✋  Note: GitHub is showing bunch of changing, mostly because I re-ordered code structure, bringing the printing detail from bottom up to this line and so on.